core/src/main/java/org/elasticsearch/bootstrap/ESPolicy.java

```diff
@@ -81,10 +83,39 @@ public boolean implies(ProtectionDomain domain, Permission permission) {
             }
         }
 
+        // Special handling for broken Hadoop code: "let me execute or my classes will not load"
+        // yeah right, REMOVE THIS when hadoop is fixed
+        if (permission instanceof FilePermission && "<<ALL FILES>>".equals(permission.getName())) {
```

Comment:
this permission is only requested by hadoop, it is the only thing really trying to execute like that.